src/deploy/functions/cache/hash.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from "fs";
+import * as crypto from "crypto";
+import * as args from "../args";
+import * as secrets from "../../../functions/secrets";
+import * as backend from "../backend";
+
+export async function getLocalHash(source: args.Source, wantBackend: backend.Backend) {
```

Comment:
Make this take an exact source file and then call the function twice: once for sourceV1 and once for sourceV2. The former may contain a dynamically generated runtimeconfig.json and the latter definitely won't.

Reply:
I don't quite follow. I'll ping you offline.

Reply:
Spoke offline, I understand now. Updated the signature.